student-os/scripts/repair_import_apply.py

```
@@ -0,0 +1,171 @@
+    payload = json.loads(path.read_text(encoding="utf-8"))
+    if not isinstance(payload, dict):
+        raise SystemExit(f"Review JSON must be an object: {type(payload).__name__}")
+    if payload.get("schema_version") != REVIEW_SCHEMA_VERSION:
+        raise SystemExit(f"Unsupported review schema_version: {payload.get('schema_version')}")
+    return payload
+
+
+def is_relative_to(path: Path, root: Path) -> bool:
+    try:
+        path.resolve().relative_to(root.resolve())
+        return True
```

Ruff (0.16.2)

[warning] Consider moving this statement to an `else` block

(TRY300)